EXAMPLES.md

```diff
@@ -0,0 +1,160 @@
+# Examples
+
+## Structure and concepts
+
+There are 3 types of config objects that you can put in your config file. They follow 2 purposes: registration and configuration.
+
+- Plugin - allows you to register a plugin that provides rules to run. (We will consider this one in more detail in a later section)
+- Config Object - allows you to configure the behaviour of rules provided by the plugins. It has the following shape:
+  ```text
+  {
+  	"files"?: <GlobArray>,
+  	"ignores"?: <GlobArray>,
+  	"rules": {
+  		<RuleName>: <Severity> | [<Severity>, <RuleOptions>]
+  	}
+  }
+  ```
+  ```javascript
+  export default defineConfig({
+    files: ['./src/shared/**'],
+    ignores: ['./src/shared/__mocks__/**'],
+    rules: {
+      'fsd/no-public-api': 'off',
+      'fsd/forbidden-imports': ['warn', { someOption: false }],
+    },
+  })
+  ```
+- Global ignore - allows you to disable all rules for a specific part of the file system.
+  ```text
+  {
+  	ignores: <GlobArray>
+  }
+  ```
+  ```javascript
+  export default defineConfig([
+    ...fsd.configs.recommended,
+    {
+      ignores: ['./src/shared/__mocks__/**'],
+    },
+  ])
+  ```
+
+Parts of the config object:
+
+- Severity - can be one of the following: "off", "warn", "error"
+- GlobArray - string array with glob patterns to match files and folders in your project.
+- RuleOptions - an object that contains specific rule options and can be passed to the rule to configure its behavior.
```

Comment:
suggestion: honestly, I'd love if we just deleted this entire chunk of text. The file is called examples, people learn wonderfully from examples, let's just throw a bunch of diverse examples at them and let them figure it out

and if they don't, we'll just revise the examples instead of giving them the technical readout

Reply:
Okay, I'll stash this for now somewhere. It might come in handy a bit when it's time to create a document site